main.go

```diff
@@ -690,3 +735,109 @@ func reverseBytes(b []byte) []byte {
 
 	return reversed
 }
+
+// See: https://github.com/waku-org/go-waku/blob/master/examples/basic-light-client/main.go
+// Note that this requires a running waku node with lightpush enabled at localhost.
+func SendMessage(cfg *Config, membershipFile string, message string) error {
+	clusterId := uint16(100)
+	peerMulti := "/ip4/127.0.0.1/tcp/60000/p2p/16Uiu2HAkxTGJRgkCxgMDH4A4QBvw3q462BRkVJaPF5KQWkc1t4cp"
+	pubsubTopic := "/waku/2/rs/100/0"
+
+	wakuNode, err := node.New(node.WithLightPush(), node.WithClusterID(clusterId))
```

Comment:
*I think* it's not necessary to use `node.WithLightPush()` since this mounts the server side of the protocol, but here you're only acting as a client.